fdbserver/storageserver.actor.cpp

```diff
@@ -4627,6 +4652,9 @@ void changeServerKeys(StorageServer* data,
 		MutationRef clearRange(MutationRef::ClearRange, range.begin, range.end);
 		data->addMutation(data->data().getLatestVersion(), true, clearRange, range, data->updateEagerReads);
 		data->newestAvailableVersion.insert(range, latestVersion);
+		data->storage.disposeShard(range);
```

Comment:
Do we have data that needs to be deleted here or we do it to avoid potential conflicts?